source/shared/net/desktop-wallet/install-ledger-app.rst

```diff
@@ -162,37 +162,17 @@ Install Concordium Ledger app on macOS
 Install Homebrew, Python3, and pip
 ----------------------------------
 
-#. Open the Terminal application. Click the **Launchpad** icon in the **Dock**, type *Terminal* in the search field, then select **Terminal**.
+#. To open the Terminal application, click the **Launchpad** icon in the **Dock**, type *Terminal* in the search field, then select **Terminal**.
 
-#. Install the package manager Homebrew. Copy the following line into the Terminal and press enter.
+#. Navigate to where you have downloaded the Ledger install package. For example, this might be ``~/Downloads/concordium-ledger-app-2.0.1-target-2.0.0``. To navigate to this directory in a Terminal, enter ``cd ~/Downloads/concordium-ledger-app-2.0.1-target-2.0.0``.
 
-   .. code-block:: console
-
-      /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
```

Comment:
They still need to this script. That is how you download and install Homebrew :)

Reply:
So two steps seem reasonable.
1. Install homebrew
2. Install all the rest